activiti-services/activiti-services-identity-keycloak/src/main/java/org/activiti/services/identity/keycloak/SecurityConfig.java

```diff
@@ -74,6 +74,6 @@ protected void configure(HttpSecurity http) throws Exception {
         super.configure(http);
         http.authorizeRequests()
                 .antMatchers("/*").hasRole("user") //could optionally set these by autowiring and parsing keycloakSpringBootProperties
```

Comment:
@ryandawsonuk here we are adding another place to configure roles and what they are allowed to do?